src/Analysis/GWASResultsV2/Views/Home/HomeTable/HomeTable.jsx

```diff
@@ -1,94 +1,295 @@
-import React, { useContext } from 'react';
-import { Button, Table, Space } from 'antd';
+import React, { useContext, useEffect, useState } from 'react';
+import {
+  Button, Table, Space, Input, DatePicker, Select,
+} from 'antd';
+import { SearchOutlined } from '@ant-design/icons';
 import PropTypes from 'prop-types';
+import moment from 'moment';
 import SharedContext from '../../../Utils/SharedContext';
 import ActionsDropdown from './ActionsDropdown/ActionsDropdown';
 import Icons from './TableIcons/Icons';
 import DateForTable from '../../../SharedComponents/DateForTable/DateForTable';
 import PHASES from '../../../Utils/PhasesEnumeration';
+import filterTableData from './filterTableData';
+import VIEWS from '../../../Utils/ViewsEnumeration';
 import './HomeTable.css';
 
+const { RangePicker } = DatePicker;
+
 const HomeTable = ({ data }) => {
-  const { setCurrentView, setSelectedRowData } = useContext(SharedContext);
+  const {
+    setCurrentView,
+    setSelectedRowData,
+    homeTableState,
+    setHomeTableState,
+  } = useContext(SharedContext);
+
+  const handleTableChange = (pagination, filters, sorter) => {
```

Comment:
`filters` not used here. Remove?

Reply:
No. This is a sent as part of the event, removing breaks sorting because then the sorter object wouldn't be in the right order. It could be renamed if desired to ``` _ ``` for example, but I think it makes sense to keep it here so we know what's being sent to the function

Reply:
ok, got it